components/org.wso2.carbon.identity.scim.common/pom.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
+<?xml version="1.0" encoding="UTF-8"?><!--
```

Comment:
Shouldn't the comment starting tag be starting from a new line? Check and fix in all the places

Reply:
Fixed in db4763aec62900ad194b54c662e56c31b66f2575.